.circleci/config.yml

```diff
@@ -5,73 +5,22 @@
-    ####   /go/src/github.com/circleci/go-tool
-    ####   /go/src/bitbucket.org/circleci/go-tool
-    working_directory: /go/src/github.com/sul-dlss-labs/sparql-loader
-    steps:
-      - checkout
-
-      # specify any bash command here prefixed with `run: `
-      - run: go get -v -t -d ./...
-      - run: go get github.com/golang/dep/cmd/dep
-      - run: dep ensure
-      - run: go test -v ./... -short
-      - run: go build -o neptune cmd/neptune/main.go
-
-  deploy:
-    docker:
-      - image: circleci/golang:latest
-    working_directory: /go/src/github.com/sul-dlss-labs/sparql-loader
+      - image: circleci/python:3.6.4  # primary container
     steps:
+      - checkout # checkout source code to the working directory
+      - run: sudo chown -R circleci:circleci /usr/local/bin
+      - run: sudo chown -R circleci:circleci /usr/local/lib/python3.6/site-packages
       - run:
-          name: Install AWS CLI
-          command: |
-            sudo apt-get install -y python-pip libyaml-dev python-dev jq
-            sudo pip install awscli
-      - checkout
-      - run: go get github.com/golang/dep/cmd/dep
-      - run: dep ensure
-      - run: GOOS=linux go build -o neptune cmd/neptune/main.go
-      - run: zip lambda.zip neptune
-      - run:
-          name: Upload to S3
           command: |
-            mkdir ~/.aws
-            echo -e "[rialto]\naws_access_key_id=$CIRCLE_ACCESS_KEY_ID\naws_secret_access_key=$CIRCLE_SECRET_KEY\n" > ~/.aws/credentials
-            unset  AWS_SESSION_TOKEN
-            temp_role=$(aws sts assume-role \
-                  --role-session-name "DevelopersRole" \
-                  --role-arn $DEV_ROLE_ARN \
-                  --profile rialto)
-            export AWS_ACCESS_KEY_ID=$(echo $temp_role | jq .Credentials.AccessKeyId | xargs)
-            export AWS_SECRET_ACCESS_KEY=$(echo $temp_role | jq .Credentials.SecretAccessKey | xargs)
-            export AWS_SESSION_TOKEN=$(echo $temp_role | jq .Credentials.SessionToken | xargs)
-            aws configure set region $AWS_REGION
-            aws configure set default.output json
-            aws configure list  # Get confirmation it worked in your logs
-            aws lambda update-function-code --function-name rialto-sparql-loader-development --zip-file fileb://lambda.zip
-
-workflows:
-  version: 2
-  
-  deploy-dev:
-    jobs:
-      - build:
-          filters:
-            branches:
-              ignore: master
-      - deploy:
-          filters:
-            branches:
-              only: master
\ No newline at end of file
+            sudo pip install --upgrade pip
+            # pip3 install -r requirements.txt
+            python -m venv env
+            source env/bin/activate # load our virtual env
+            pip install rdflib
+            pip install boto3
+            pip install rdflib-sparql
+            pip install honeybadger
+            pip install pytest
+            pytest -k unit # only run unit tests at this stage
```

Comment:
Also, should add checking for pep8 compliance.

Reply:
Done. As pycodestyle now per deprecation warning